docs/docs/benchmarks/README.md

```diff
@@ -6,27 +6,28 @@ permalink: /docs/benchmarks/
 
 ## Current Benchmarks
 
-You can see the current Helios benchmarks [here](https://github.com/47deg/helios/tree/master/benchmarks/master_benchmark.json)
+You can see the current **Helios** benchmarks [here](https://github.com/47deg/helios/tree/master/benchmarks/master_benchmark.json)
 
 <fortyseven-robeen data-url="https://raw.githubusercontent.com/47deg/helios/master/benchmarks/master_benchmark.json"></fortyseven-robeen>
 
 ## Comparing Helios
 
 We have [here](https://github.com/47deg/helios/tree/master/benchmarks/libraries_benchmark.json) a benchmark comparison
-between Helios and some of the most common Kotlin Json libraries.
+between **Helios** and some of the most common `Kotlin` `Json` libraries.
 
 <fortyseven-robeen data-url="https://raw.githubusercontent.com/47deg/helios/master/benchmarks/libraries_benchmark.json"></fortyseven-robeen>
 
+The benchmark comparison is graphically showed by [**Robeen**](https://github.com/47deg/robeen) and provided by [**Hood**](https://github.com/47deg/hood).
```

Comment:
I'd probably say `rendered`, but probably nothing wrong on the use of `show`, just personal preference.